packaging/dist/ember-intl.js

```diff
@@ -155,7 +155,7 @@ var define, requireModule, require, requirejs;
         messages: {},
```

Comment:
Ignore this file, it's generated and I need to rerun the script once 2.0 is final